boards/arm/mimxrt1010_evk/mimxrt1010_evk.dts

```diff
@@ -67,3 +67,11 @@ arduino_serial: &lpuart1 {};
 &usb1 {
 	status = "okay";
 };
+
+&gpt1 {
+       nxp,gptfreq = <2500000>;
+};
+
+&gpt2 {
+       nxp,gptfreq = <2500000>;
+};
```

Comment:
Should these be set at the SoC level instead of the board level?

Reply:
ok, move to SoC level.